types/react/index.d.ts

```diff
@@ -322,7 +322,7 @@ declare namespace React {
 
     type SFC<P = {}> = StatelessComponent<P>;
     interface StatelessComponent<P = {}> {
-        (props: P & { children?: ReactNode }, context?: any): ReactElement<any> | null;
+        (props: P & { children?: ReactNode }, context?: any): ReactNode;
```

Comment:
This is slightly wrong because it can't return `undefined`. `ReactNode` definition includes undefined.

Reply:
Well, in that case, it's wrong altogether, because `ReactNode` is also used as a return type of `render` method for a class component.

Reply:
Fair enough.